src/lang/lwc_en.properties

```diff
@@ -598,3 +598,9 @@ display=Display
 prefix=%reset%
 
 lwc.admin.denied=%dark_red%You are not an LWC admin!
+
+protection.info=%dark_green%Protection ID%gray%: %reset%%id% (Type: %type%) \n\
+%yellow%  Owner%gray%: %reset%%name% (%owner%) \n\
+%yellow%  Location%gray%: %reset%%world% @ %x%, %y%, %z% \n\
+%yellow%  Created%gray%: %reset%%creation% (%lastAccessed% ago) \n\
```

Comment:
Last accessed is different from created, but you are putting it next to the text that says "Created" which may confuse people.